src/main/webapp/js/user-page-loader.js

```diff
@@ -104,6 +104,10 @@ function buildMessageDiv(message) {
   return messageDiv;
 }
 
+/**
+ * Gets the aboutMe data through our server
```

Comment:
1. Normal '/* */' is fine here since this is javascript.
2. This doesn't mention that the aboutMe element is actually updated with the data fetched from the server.
3. Consider using language along the lines of "Initiates a request to get the aboutMe data ..." instead of just get. The main reason for this is that the typical "get" or "getter" interpretation usually means the function returns the value you are "getting" immediately (e.g. datastore.getUser()). In this case it is important to make clear that you are initiating an asynchronous operation that requests the information instead of doing a synchronous get operation. Hopefully this makes sense.